.github/workflows/image.yml

```diff
@@ -0,0 +1,32 @@
+name: Image
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    name: Image build and push
+    steps:
+    - uses: actions/checkout@v2
+    - uses: docker/setup-qemu-action@v1
+    - uses: docker/setup-buildx-action@v1
+      with:
+        driver-opts: image=moby/buildkit:v0.8-beta
+    - uses: docker/login-action@v1
+      with:
+        registry: quay.io
+        username: ${{ secrets.QUAY_USERNAME }}
+        password: ${{ secrets.QUAY_TOKEN }}
+    - uses: docker/build-push-action@v2
+      with:
+        context: .
+        file: ./Dockerfile
+        platforms: linux/amd64
+        pull: true
+        push: true
+        tags: |
+          quay.io/gatekeeper/operator:v0.0.1
```

Comment:
I'm thinking we should probably remove this tag since we don't have an official release yet.